### PR TITLE
Inputs shouldn't take input in canvas view #104

### LINKED ISSUE #104: Inputs shouldn't take input in canvas view
Clicking them or their labels should just select the component instead.
If they need a default pre-filled text, it should be done in the context menu.

- [ ] text input
- [ ] text area
- [ ] search
- [ ] number input

### PR DESCRIPTION
**Pull request by vibh1103**
## Related Tickets & Documents

- Related Issue #104 
- Closes #104 

## What type of PR is this? (check all that apply)

- [ ] Feature
- [x] Bug Fix
- [ ] Refactor
- [ ] Documentation Update

### CODE REVIEW

File: src/sdk/src/fragment-components/a-numberinput.tsx [hunk 1]
```diff
@@ -147,6 +147,9 @@ export const ANumberInput = ({
 	componentObj,
 	...rest
 }: any) => {
+	const preventClickStyle = css`
+		pointer-events: none;
+	`;
```
Comment (Akshat55) [inline src/sdk/src/fragment-components/a-numberinput.tsx:152]: Move class outside of the function. Do the same for all of the files below as well

**Review by Akshat55 — CHANGES_REQUESTED** [review]
Awesome! Tested it and it works, just a few things before we can merge.

1. You need to sign off on your commits, you can do so by include `-s` flag when making the commit. You can fix your previous commits with the following commands:
    1.  Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
    2. In your local branch, run: git rebase HEAD~2 --signoff
    3. Force push your changes to overwrite the branch: git push --force-with-lease origin remove_pointer_events

2. Could you move all of the class variables outside of the function? Once this is done we should be good to merge. Make sure to sign off using `-s` flag :)

File: src/sdk/src/fragment-components/a-numberinput.tsx (hunk 1 above)
Comment (vibh1103) [reply src/sdk/src/fragment-components/a-numberinput.tsx]: done

**Review by Akshat55 — APPROVED** [review]

**Review by zvonimirfras — APPROVED** [review]

**Comment by zvonimirfras** [comment]
Thank you for your contribution, Vibhor! @vibh1103

**Pull request merged**